README.md

```diff
@@ -27,13 +27,13 @@ lintChecks "com.swvl.lint:linta-android:x.y.z"
 
 The following are the lint checks currently implemented by Linta [please add the documentation to any recent addition and/or any missing ones]:
 
-| Lint Issue ID           |    Severity   | Description                                                                                            |
-|:-----------------------:|:-------------:|--------------------------------------------------------------------------------------------------------|
-| `DuplicateColors`       |    warning    | When a duplicate color is defined in a resource file                                                   |
-| `DuplicateResourceFiles`|    warning    | When there are two or more duplicate resource files containing the same exact attributes, regardless of differences in whitespaces, attributes order, or used tools namespace if any |
-| `HardcodedColorSrcCode` |     error     | When a hardcoded color is used in a source code file (Java or Kotlin)                                  |
-| `HardcodedColorXml`     |     error     | When a hardcoded color is used in a resource file (drawables, layouts, etc.)                           |
-| `RedundantStyles`       |    warning    | When a style is created without adding any new attributes                                              |
+| Lint Issue ID           |    Severity   | Description                                                                                                                                                                                   |
+|:-----------------------:|:-------------:|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `DuplicateColors`       |    warning    | When a duplicate color is defined in a resource file                                                                                                                                          |
+| `DuplicateResourceFiles`|    warning    | When there are two or more duplicate resource files containing the same exact attributes, regardless of differences in whitespaces, attributes order, comments or used tools namespace if any |
```

Comment:
I would prefer to add a comma before "or" when "or" is used in an expression of more than two items, since the two items are unrelated to each other in the sentence.

- **Source:** [Grammarly](https://www.grammarly.com/blog/comma-before-or/)

Please make sure to also make the last column in all rows have the same width after adding the comma. :)

```suggestion
| `DuplicateResourceFiles`|    warning    | When there are two or more duplicate resource files containing the same exact attributes, regardless of differences in whitespaces, attributes order, comments, or used tools namespace if any |
```

Reply:
where did you get that? 😄 

> Should you use a comma before or? The answer depends on how you are using or. Always place a comma before or when it begins an independent clause, but if it begins a dependent clause, don’t. In a series (or list) of three or more items, you can use a comma before or, but this is a preference, not a rule.

> You may want to use a comma in a series of three or more things. This is the much-debated[ Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) (or serial comma). Although the Oxford University Press uses it (hence the name), [American English](https://www.grammarly.com/blog/how-british-english-and-american-english-are-different/) writers use it more often than British English writers. Whether or not you should use it is a stylistic choice

Reply:
From the references you mentioned. It's a preference as I mentioned, but I find it makes more sense than without a comma here. :)

Reply:
why tho? what value does it add? To me it looks odd, because to me the "or" does this job of separation so the comma just feels redundant. I also don't recall seeing it much in formal writings